manifest/armv7l/c/crew_sudo.filelist

```diff
@@ -0,0 +1,59 @@
+/usr/local/bin/crew-sudo
+/usr/local/bin/sudo
+/usr/local/etc/bash.d/crew_sudo
+/usr/local/etc/env.d/crew_sudo
+/usr/local/lib/crew-sudo/.git/FETCH_HEAD
+/usr/local/lib/crew-sudo/.git/HEAD
+/usr/local/lib/crew-sudo/.git/config
+/usr/local/lib/crew-sudo/.git/description
+/usr/local/lib/crew-sudo/.git/hooks/applypatch-msg.sample
+/usr/local/lib/crew-sudo/.git/hooks/commit-msg.sample
+/usr/local/lib/crew-sudo/.git/hooks/fsmonitor-watchman.sample
+/usr/local/lib/crew-sudo/.git/hooks/post-update.sample
+/usr/local/lib/crew-sudo/.git/hooks/pre-applypatch.sample
+/usr/local/lib/crew-sudo/.git/hooks/pre-commit.sample
+/usr/local/lib/crew-sudo/.git/hooks/pre-merge-commit.sample
+/usr/local/lib/crew-sudo/.git/hooks/pre-push.sample
+/usr/local/lib/crew-sudo/.git/hooks/pre-rebase.sample
+/usr/local/lib/crew-sudo/.git/hooks/pre-receive.sample
+/usr/local/lib/crew-sudo/.git/hooks/prepare-commit-msg.sample
+/usr/local/lib/crew-sudo/.git/hooks/push-to-checkout.sample
+/usr/local/lib/crew-sudo/.git/hooks/sendemail-validate.sample
+/usr/local/lib/crew-sudo/.git/hooks/update.sample
+/usr/local/lib/crew-sudo/.git/index
+/usr/local/lib/crew-sudo/.git/info/exclude
+/usr/local/lib/crew-sudo/.git/logs/HEAD
+/usr/local/lib/crew-sudo/.git/objects/0d/70b59b82473d804742b579580cc736c8b54a8e
+/usr/local/lib/crew-sudo/.git/objects/0d/a27e544de02029a1b09a76663714570d6baf96
+/usr/local/lib/crew-sudo/.git/objects/18/fff49e83334621d50b6b9a8ff2acbe2e5aaf3b
+/usr/local/lib/crew-sudo/.git/objects/1a/ebe6702608199019d28d19eb2c7c28543324ab
+/usr/local/lib/crew-sudo/.git/objects/1c/dcbd315e42c98c7619b30e4c55c08dad923fb3
+/usr/local/lib/crew-sudo/.git/objects/2f/bfeb09534d40b6f8b7dc3c84b07013fc574ed7
+/usr/local/lib/crew-sudo/.git/objects/56/e17f6a8fcda875f71e894e0dd126f39c605cbf
+/usr/local/lib/crew-sudo/.git/objects/63/589107850c597f960a1bca6b5de1de37f91e18
+/usr/local/lib/crew-sudo/.git/objects/63/f7a6666d70ab4d5b8d83c7d6d0f4774e7086a1
+/usr/local/lib/crew-sudo/.git/objects/6b/303bd1ef3ef2ac67c5c3b8025b81b4ee871cce
+/usr/local/lib/crew-sudo/.git/objects/90/129d51efc3003b71d243222e4b9c56d91012a9
+/usr/local/lib/crew-sudo/.git/objects/9b/08247f6646266d482672f074fcd8048a81f000
+/usr/local/lib/crew-sudo/.git/objects/aa/0b8edd03cbe3ed4954c6f48ae5619b3071b0c5
+/usr/local/lib/crew-sudo/.git/objects/bb/1aa3cad22c877af8357be686f44e4aa07420ed
+/usr/local/lib/crew-sudo/.git/objects/bc/5d987d52abadbe69ceff8d622ff5a22a7c0a69
+/usr/local/lib/crew-sudo/.git/objects/c4/ab9ed0894d37355cdf6fea1aa7bd6d8f49bd6c
+/usr/local/lib/crew-sudo/.git/objects/ec/05ac75a5033312bcb29d0a815663689240737a
+/usr/local/lib/crew-sudo/.git/objects/f2/88702d2fa16d3cdf0035b15a9fcbc552cd88e7
+/usr/local/lib/crew-sudo/.git/objects/f2/98aed20e064e68611ed0160ee0d993ffe26753
+/usr/local/lib/crew-sudo/.git/objects/f8/d717d879ee3fc1e26b3e7a0e6dbdabfa0fef13
+/usr/local/lib/crew-sudo/.git/shallow
```

Comment:
We wouldn't need the .git directory if we used the source download to build, right?

Reply:
We should modify that to remove that... let me do that...